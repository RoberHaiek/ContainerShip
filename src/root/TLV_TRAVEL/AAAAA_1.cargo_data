#<uid>,<weight>,<dstPort>
container1,1,AAAAA
container2,1,BBBBB
container3,1,CCCCC
container4,1,DDDDD