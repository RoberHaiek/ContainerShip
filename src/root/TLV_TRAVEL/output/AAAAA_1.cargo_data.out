container-1,load,1,1,1
container-2,load,1,1,1
container-3,load,1,1,1
container-4,load,1,1,1