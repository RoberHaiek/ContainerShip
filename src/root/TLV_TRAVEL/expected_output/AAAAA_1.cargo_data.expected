container1,load,1,1,1
container2,load,1,1,1
container3,load,1,1,1
container4,load,1,1,1